### ISSUE TITLE
routing issue: пр. Энгельса-Выборгское ш.

### ISSUE THREAD

**Issue by 532910** [osmandapp/OsmAnd#10592]
https://www.openstreetmap.org/directions?engine=fossgis_osrm_car&route=60.02828%2C30.31948%3B60.04072%2C30.31718#map=15/60.0349/30.3237

![image](https://user-images.githubusercontent.com/14617699/104807440-35b8c080-57f0-11eb-956b-2f6a28e3b88d.png)
![image](https://user-images.githubusercontent.com/14617699/104807461-54b75280-57f0-11eb-9420-62a053ba25d2.png)

v3.8.5 from F-Droid

**Comment by ManDay** [comment]
Just from looking at the route, it may want to bypass the [traffic light](https://www.openstreetmap.org/node/1566538124)

**Comment by vshcherb** [comment]
Yes this is quite many traffic lights which are better to have not on intersection.
Also this road https://www.openstreetmap.org/way/134019650 might be a primary_link
